database.json

```diff
@@ -12,5 +12,92 @@
+                    "Use the same log search as before"
+                ]
+            },
+            {
+                "Question:":"What is the correlationVector for the affected request?",
+                "Answer": [
+                    "m7TcM/K6QESTUAP/jbvF+w",
+                    "Nu/I3T46IUG8yOGa65KS7A",
+                    "noy8vmqf9EaEDKyM2nIS0w",
+                    "sTZd9h0v1EqYY289qH5dbw",
+                    "ToKjJdZ1C0G+DBCh0pawEg",
+                    "EJbMMJJW0EyB8rhqRFIUiQ",
+                    "lkGUj5oi+0Ct4AZRTRirQQ",
+                    "KXQxfJZAAEmrRn6h0OHLpg"
+                ],
+                "Hints": [
+                    "This information can be found in the logs",
+                    "Use the same log search as before"
+                ]
+            },
+            {
+                "Question:":"What was the underlying error message for the failed request?",
+                "Answer": "The Resource Provider returned '499' status code.",
+                "Hints": [
+                    "Search the logs for a specific correlation Id/vector",
+                    "Make sure you remove the client query as that will scope your results down too far",
+                    "The underlying error message is often encrypted. See https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/20933/Decrypting-SMS-Logs",
+                ]
+            },
+            {
+                "Question:":"Is this an ongoing issue, or was it transient?",
+                "Answer": "Transient",
+                "Hints": [
+                    "Was the issue auto-mitigated?",
+                    "What is the volume of errors and have we seen similar errors over time?"
+                ]
+            },
+            {
+                "Question:":"How many customers were affected by this incident?",
+                "Answer": "1",
+                "Hints": [
+                    "Refer to this TSG: https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/19428/TSG-Gathering-Impacted-Resource-Information-From-Logs",
+                    "Since it's an issue with Resource Provider we don't have access to the resource Id, but is there anything else that could identify the caller?"
+                ]
+            },
+            {
+                "Question:":"Has this issue occurred in the recent past? Is there an ICM already describing this issue?",
+                "Answer": "This is why we should be keeping our ICM queue cleaner.  Refer to this wiki for instructions on parenting an ICM and/or creating repair items: https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/12707/SMS-On-Call-Onboarding?anchor=during-secondary-on-call",
+                "Hints": [
+                    "Look in the ICM queue for any similar issues",
+                    "Keep in mind issues with downstream dependencies like Resource Provider can occur across endpoints, so don't just look for ICMs with the exact same title as the current alert.",
```

Comment:
The comma at the end of the list should be removed, after "title as the current alert."